src/sql-parser/tests/testdata/ddl

```diff
@@ -1781,6 +1781,20 @@ GRANT scientist TO joseph, michael
 =>
 GrantRole(GrantRoleStatement { role_name: Ident("scientist"), member_names: [Ident("joseph"), Ident("michael")] })
 
+parse-statement
+GRANT scientist TO GROUP joseph, GROUP michael
```

Comment:
would you mind adding one more test case that contains three groups? e.g.
```
GRANT scientist TO GROUP joseph, GROUP michael, GROUP john
```

Reply:
Done